src/main/java/org/eln2/mc/common/network/Networking.kt

```diff
@@ -24,19 +28,47 @@ object Networking {
     private val channel = NetworkRegistry.newSimpleChannel(
         ResourceLocation(Eln2.MODID, channelName),
         { protocolVersion },
-        {it == protocolVersion },
-        {it == protocolVersion })
+        { it == protocolVersion },
+        { it == protocolVersion })
 
 
-    fun setup(){
-        channel.registerMessage(PacketType.CIRCUIT_EXPLORER_OPEN_PACKET.id, CEOPacket::class.java, CEOPacket::encode, ::CEOPacket, CEOPacket::handle)
-        channel.registerMessage(PacketType.CIRCUIT_EXPLORER_CONTEXT_PACKET.id, CECPacket::class.java, CECPacket::encode, ::CECPacket, CECPacket::handle)
+    fun setup() {
+        Eln2.LOGGER.info("Registering network packets")
+        channel.registerMessage(
+            PacketType.CIRCUIT_EXPLORER_OPEN_PACKET.id,
+            CEOPacket::class.java,
+            CEOPacket::encode,
+            ::CEOPacket,
+            CEOPacket::handle
+        )
+        channel.registerMessage(
+            PacketType.CIRCUIT_EXPLORER_CONTEXT_PACKET.id,
+            CECPacket::class.java,
+            CECPacket::encode,
+            ::CECPacket,
+            CECPacket::handle
+        )
```

Comment:
Since we've got the other ones at regular length and all indented, might as well remove the typealias at the top of the file.